### PR TITLE
Change the results look & feel a little bit

### PR DESCRIPTION
**Pull request by fzxu**
* Make the entity detail resizable
* Move the result table out of the frame
* Change the result page look & feel a little bit...
* Theme is temporary, feel free to make it prettier

### CODE REVIEW

File: app/src/components/entityDetails/EntityDetails.js [hunk 1]
```diff
@@ -1,22 +1,35 @@
 import React, { Component } from 'react';
+import { withStyles } from '@material-ui/core/styles';
```
[the remark this answers is not in the document]
Comment (fzxu) [reply app/src/components/entityDetails/EntityDetails.js]: sure. I will reorder the imports.

File: app/src/components/results/Results.js [hunk 2]
```diff
@@ -2,47 +2,28 @@ import React, { Component } from 'react';
...
-import TableRow from '@material-ui/core/TableRow';
-import Paper from '@material-ui/core/Paper';
 import CircularProgress from '@material-ui/core/CircularProgress';
 import Typography from '@material-ui/core/Typography';
 
 import EntityDetails from '../entityDetails/EntityDetails';
 import * as apiCfg from '../../config/apiConfig';
 import * as queryActions from '../../actions/queryActions';
-import './Results.css';
+import SplitterLayout from 'react-splitter-layout';
```
[the remark this answers is not in the document]
Comment (fzxu) [reply app/src/components/results/Results.js]: will do

File: app/src/components/results/Results.js [hunk 3]
```diff
@@ -87,68 +68,24 @@ class Results extends Component {
   //TODO:DM - should we extract table construction into ResultList comp?
   render() {
     const { classes, query } = this.props;
+    const { selectedIdx } = this.state;
+
     return (
       <div className='Results'>
+        <Typography variant="h6" gutterBottom className={classes.resultTitle}>
+          Search Result: {query.current}
```
[the remark this answers is not in the document]
Comment (fzxu) [reply app/src/components/results/Results.js]: yeah, for me the search box in the menubar on top right is not obvious to know what is showing...

File: app/src/index.js [hunk 4]
```diff
@@ -19,20 +19,122 @@ WebFont.load({
...
+      fontWeight: 500,
+      fontSize: 26,
+      letterSpacing: 0.5,
+    },
+  },
   palette: {
     primary: {
-      main: '#242321',
-    },
-    secondary: {
```
[the remark this answers is not in the document]
Comment (fzxu) [reply app/src/index.js]: sure I will keep the secondary.

File: app/src/index.js [hunk 5]
```diff
@@ -19,20 +19,122 @@ WebFont.load({
...
     },
   },
-  typography: {
-    useNextVariants: true,
+  shape: {
+    borderRadius: 8,
   },
 });
 
+theme = {
```
[the remark this answers is not in the document]
Comment (fzxu) [reply app/src/index.js]: I copied from one of the free theme... forgot which one it was... I don't expected this to be kept long though. feel free to change or remove....

File: app/src/components/results/Results.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (davemasselink) [reply app/src/components/results/Results.js]: glad you highlighted as potential UX issue

**Pull request merged**